### ISSUE TITLE
Add strategy management script

### ISSUE THREAD

**Issue by fubuloubu** [yearn/yearn-vaults#18]
- [ ] Add a strategy (by address, using pre-defined params w/ overrides available)
- [ ] Update params for strategy (single parameter or multiple params at once)
- [ ] Revoke a strategy
- [ ] Manage withdrawal queue (reordering, adding, removing)
- [ ] Allow "batching" multiple actions together using MultiCall
- [ ] Add a method to "delegate to" another account for execution (such as Gnosis Safe)

**Comment by fameal** [comment]
Is it superseded by CMO? Can we close this?

**Comment by fubuloubu** [comment]
> Is it superseded by CMO? Can we close this?

not really, could complement CMO actually if it were published as a package separately that we could use in a management script, which could be used to deliver transactions to multisig
